libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/MapRouteLine.kt

```diff
@@ -420,22 +422,42 @@ internal class MapRouteLine(
      * @param directionsRoutes the routes to be represented on the map.
      */
     fun draw(directionsRoutes: List<DirectionsRoute>) {
-        reinitializeWithRoutes(directionsRoutes)
+        val featureDataProvider: () -> List<RouteFeatureData> =
+            getRouteFeatureDataProvider(directionsRoutes)
+        reinitializeWithRoutes(directionsRoutes, featureDataProvider)
+        drawRoutes(routeFeatureData)
+    }
+
+    fun drawIdentifiableRoute(directionsRoutes: IdentifiableRoute) {
+        drawIdentifiableRoutes(listOf(directionsRoutes))
+    }
+
+    fun drawIdentifiableRoutes(directionsRoutes: List<IdentifiableRoute>) {
+        val routes = directionsRoutes.map { it.route }
+        val featureDataProvider: () -> List<RouteFeatureData> =
+            getIdentifiableRouteFeatureDataProvider(directionsRoutes)
+        reinitializeWithRoutes(routes, featureDataProvider)
         drawRoutes(routeFeatureData)
     }
 
     internal fun reinitializeWithRoutes(directionsRoutes: List<DirectionsRoute>) {
```

Comment:
Does this need to be explicitly `internal`? The class itself is `internal` 🤔 

Same thing happens to `reinitializePrimaryRoute` and `getRouteColorForCongestion` 👇